litellm_config.yml

```diff
@@ -92,7 +92,8 @@ model_list:
 
   - model_name: anthropic/claude-3.5-sonnet
     litellm_params:
-      model: claude-3-5-sonnet-20240620
+      # model: claude-3-5-sonnet-20240620
+      model: claude-3-5-sonnet-20241022
```

Comment:
new model seems a bit off, it has very strong 'beliefs' about the current timeline and refuses to execute tasks if it evaluates some required date params to be in the future